lambda/install.js

```diff
@@ -29,10 +29,12 @@ function sha256sum(p) {
 }
 
 async function download(url, dest) {
-  // TODO: Support proxy download
+  const agent = {};
+  agent.https = process.env.HTTPS_PROXY ? new HttpsProxyAgent({proxy: process.env.HTTPS_PROXY}): undefined
+  agent.http = process.env.HTTP_PROXY ? new HttpProxyAgent({proxy: process.env.HTTP_PROXY}): undefined
```

Comment:
Missing tailing `;`

Reply:
Ah, thought that runinng lint was enough but it seems to be ignoring js-files.